spring-mvc-simple-2/src/main/java/com/baeldung/exclude_urls_filter/filter/HeaderContextFilter.java

```diff
@@ -0,0 +1,27 @@
+package com.baeldung.exclude_urls_filter.filter;
+
+import com.baeldung.exclude_urls_filter.utils.HeaderContextProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.core.annotation.Order;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+
+@Order(1)
+public class HeaderContextFilter implements Filter {
+
+	@Override
```

Comment:
fix

Reply:
^ @rpvilao: I didn't understand what needs to be fixed here. Though, I've made changes in the same piece of code as per your other comments.
Please let me know if it still needs any change.